client/domain/QuestionObjectFactorySpec.js

```diff
@@ -78,6 +85,10 @@ describe('QuestionObjectFactory', function() {
       expect(question.isLastTask(2)).toBe(false);
       expect(question.isLastTask(-1)).toBe(false);
       expect(question.isLastTask(1)).toBe(true);
+      expect(question.isLastTask(question.getTasks.length -
```

Comment:
I corrected that in a different PR. I'll look into it and see if I can find out why it's doing this.